### PR TITLE
Summarizing where we are

### PR DESCRIPTION
**Pull request by gvwilson**
I needed to summarize the state of play for a potential backer, and thought I'd turn it into a blog post that people could point to when asked, "What is this anyway?"

### CODE REVIEW

**Comment by ahmadia** [comment]
This looks very good.  Thanks for the update.

**Comment by wking** [comment]
On Thu, May 16, 2013 at 12:47:56PM -0700, Greg Wilson wrote:

> - Summarizing where we are

In general this looks good to me.  It overlaps with the (stalled?)
lessons-learned paper, but I don't think we can bang on this drum too
hard ;).

You have “but the "what" for scientific software is still very much up
in the air”.  Did you mean “the "why"”?  It's hard to imagine
understanding “how” without understanding “what” ;).

I think “by watching us work, people learn more than we realized…”
would read more cleanly.  I initially parsed it as “we were teaching
them by watching us work”, which is too introspective ;).

Again, you have “how” and “what”.  Maybe I'm missunderstanding what
“what” is :p.  I imagine “what” is the syllabus you list, “how” is
what the instructor does to teach it, and “why” is why the students
should show up ;).  I expect scientist instructors could more
naturally put “why” in context than non-scientist instructors (or at
least give that impression to students who were thinking of signing
up).  I don't see their angle on “how”…

`&hellip;` ;).

Links to the assessments?

In the data management paragraph, I'd replace “they” with “students”.

'nother `&hellip;`.

In the DiRAC paragraph, “want to use it” should be “want to use the
facility”, because the previous “it” referenced the driver's license.

File: blog/2013/05/where-we-are.html [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+  the speedup is measured in small percentages,
+  not orders of magnitude.
+  For most scientists,
+  getting the code to do the right thing
+  is now a bigger bottleneck than its running time.
+</p>
+<p>
+  That's where we come in.
+  We teach scientists
+  (usually grad students, since they have both need and time)
```
Comment (joonro) [inline blog/2013/05/where-we-are.html]: This line seems to be missing something: they have both **need and time**

File: blog/2013/05/where-we-are.html [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+    and how to use the web to do science better;
+    the hard part is to figure out what.
+    We can talk generalities about data management,
+    but they want tools and procedures.
+    As for web programming,
+    all we can show them in half a day is
+    how to create security holes...
+  </li>
+  <li>
+    We need to persuade the powers that be&mdash;the NSF,
```
Comment (npch) [inline blog/2013/05/where-we-are.html]: Should we use the more general term "research funders" rather than the specific "NSF"?

**Comment by sprossiter** [comment]
"We teach scientists (usually grad students, since they have both need and time)
  what they ought to know before they tackle anything with "peta" or "cloud" in its name."

I like the style but not the substance of this sentence. The peta/cloud bit suggests good software skills are only important for big-data and big-computational-power problems. Don't we want this to be more inclusive?

File: blog/2013/05/where-we-are.html [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+  For most scientists,
+  getting the code to do the right thing
+  is now a bigger bottleneck than its running time.
+</p>
+<p>
+  That's where we come in.
+  We teach scientists
+  (usually grad students, since they have both need and time)
+  what they ought to know
+  before they tackle anything with "peta" or "cloud" in its name.
```
Comment (jiffyclub) [inline blog/2013/05/where-we-are.html]: I agree with others that this is exclusive. Most researchers will _never_ tackle anything with "peta" or "cloud" in its name, but they still need us.

Comment (wking) [inline blog/2013/05/where-we-are.html]: On Fri, May 17, 2013 at 05:50:44PM -0700, Matt Davis wrote:

> Most researchers will _never_ tackle anything with "peta" or "cloud"
> in its name, but they still need us.

I tackled about a million peta-molecules of theobromine for desert an
hour ago.  For a moment, I was on cloud nine ;).

File: blog/2013/05/where-we-are.html [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+  they'll be better able to take advantage of more sophisticated things.
+</p>
+<p>
+  Between January 2012 and July 2013,
+  over 100 volunteers will have run 92 two-day workshops for over 3000 scientists.
+  A typical two-day curriculum is:
+</p>
+<ul>
+  <li>
+    <strong>Day 1 a.m.</strong>:
```
Comment (jiffyclub) [inline blog/2013/05/where-we-are.html]: I'd change these to "Day 1 - Morning" and "Day 1 - Afternoon" to avoid confusion with "1 AM" and "1 PM".

**Comment by jiffyclub** [comment]
What about a concluding paragraph with a few words about where we're going? Or do we know?

**Pull request merged**